### PR TITLE
[acl] Add VLAN bindings to dataplane ACL tables

### PR DESCRIPTION
**Pull request by daall**
Signed-off-by: Danny Allen <daall@microsoft.com>

**What I did**
I added the VLAN bind option to ACL tables.

**Why I did it**
So that ACLs can be bound to VLAN ports.

**How I verified it**
Updated tests. Also validating locally.

**Details if related**

### CODE REVIEW

**Review by qiluo-msft — APPROVED** [review]

**Comment by lguohan** [comment]
can you add vstest for this? 

what does the config db entry looks like, can you share example?

File: orchagent/aclorch.cpp [hunk 1]
```diff
@@ -1332,7 +1332,7 @@ bool AclTable::create()
     }
     else
     {
-        bpoint_list = { SAI_ACL_BIND_POINT_TYPE_PORT, SAI_ACL_BIND_POINT_TYPE_LAG };
+        bpoint_list = { SAI_ACL_BIND_POINT_TYPE_PORT, SAI_ACL_BIND_POINT_TYPE_LAG, SAI_ACL_BIND_POINT_TYPE_VLAN };
```
Comment (prsunny) [inline orchagent/aclorch.cpp:1335]: will this be supported on all platforms?

Comment (daall) [reply orchagent/aclorch.cpp:1335]: I am still checking, that is why this is in draft :)

**Comment by liat-grozovik** [comment]
@daall i think you are missing tests to cover the new configuration. if this is now allowed need to add specific case for adding and deleting this type of acl. i agree that it will be better to have a description of the schema of how to configure this type of ACL. we have few examples in the swss chema md file. 

there is also a sonic.mgmt test which covers acl. will you have a seperated PR for sonic.mgmt to extend the test configuration and check functionality?

**Pull request closed without merging**